include/cglm/types-struct.h

```diff
@@ -33,4 +33,62 @@ typedef union CGLM_ALIGN_IF(16) vec4s {
   vec4 raw;
 } vec4s;
 
+typedef vec4s versors;
+
+typedef union mat3s {
+#ifndef CGLM_NO_ANONYMOUS_STRUCT
+  struct {
+    float m00;
+    float m01;
+    float m02;
+    float m10;
+    float m11;
+    float m12;
+    float m20;
+    float m21;
+    float m22;
+  };
+  struct {
+    vec3s col0;
+    vec3s col1;
+    vec3s col2;
+  };
+#endif
+  mat3 raw;
+} mat3s;
+
+#ifdef __AVX__
+typedef union CGLM_ALIGN_IF(32) mat4s {
+#else
```

Comment:
Can you use `CGLM_ALIGN_MAT` instead of `CGLM_ALIGN_IF`. `CGLM_ALIGN_MAT` already checks these.

Reply:
Yes, sounds good.

Reply:
Making this change I notice that in the `types.h` file:
```c
#ifdef __AVX__
#  define CGLM_ALIGN_MAT CGLM_ALIGN(32)
#else
#  define CGLM_ALIGN_MAT CGLM_ALIGN(16)
#endif
```
and later `mat4` is defined like this:
```c
#ifdef __AVX__
  typedef CGLM_ALIGN_IF(32) vec4  mat4[4];
#else
  typedef CGLM_ALIGN_IF(16) vec4  mat4[4];
#endif
```
shouldn't that just be:
```c
typedef CGLM_ALIGN_MAT vec4  mat4[4];
```
`CGLM_ALIGN_MAT` is defined in terms of `CGLM_ALIGN` and not `CGLM_ALIGN_IF`, so I could be wrong here.

Reply:
Good catch. I have updated in on master.